### PR TITLE
roadmap: create ROADMAP.md

### PR DESCRIPTION
**Pull request by queenypingcap**
## What have you changed? (mandatory)
I have copied the TiKV roadmap to this repo so that TiKV can have its own roadmap

## What are the type of the changes? (mandatory)

Improvement

## How has this PR been tested? (mandatory)

N/A

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

## Refer to a related PR or issue link (optional)

https://github.com/pingcap/docs/pull/525 

## Benchmark result if necessary (optional)

No.

## Add a few positive/negative examples (optional)

N/A

### CODE REVIEW

**Comment by breeswish** [comment]
@siddontang PTAL

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - [ ] Raft Engine
+- [x] RocksDB 
+    - [x] DeleteRange
+    - [ ] BlobDB 
+- [x] Transaction
+    - [x] Optimize transaction conflicts
+    - [ ] Distributed GC
+- [x] Coprocessor
+    - [x] Streaming
+- [ ] Tool
```
Comment (Hoverbear) [inline ROADMAP.md]: Can you please add the TiKV client (Rust crate) here? :)

Comment (queenypingcap) [reply ROADMAP.md]: @siddontang can we?

Comment (siddontang) [reply ROADMAP.md]: yes

**Comment by siddontang** [comment]
maybe we should create a docs folder and move ROADMAP.md to it.

**Comment by queenypingcap** [comment]
Close this PR because the content is added in https://github.com/pingcap/tikv/pull/3301/files

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- [x] Transaction
+    - [x] Optimize transaction conflicts
+    - [ ] Distributed GC
+- [x] Coprocessor
+    - [x] Streaming
+- [ ] Tool
+    - [x] Import distributed data
+    - [ ] Export distributed data
+    - [ ] Disaster Recovery
+    - [ ] TiKV client (Rust crate)
```
Comment (Hoverbear) [inline docs/ROADMAP.md]: It's here twice now :(

**Pull request closed without merging**